include/RandLAPACK/drivers/rsvd.hh

```diff
@@ -1,9 +1,8 @@
-#ifndef BLAS_HH
-#include <blas.hh>
-#define BLAS_HH
-#endif
+#ifndef randlapack_drivers_rsvd_h
+#define randlapack_drivers_rsvd_h
 
 #include "../comps/qb.hh"
+#include "blaspp.h"
```

Comment:
I'm confused about these two lines. The first line seems to imply that include statements with quotations are relative to the position of the current file. But if that's true, then the second include statement should be ``#include "../blaspp.h``. Is the current format really valid? If so, can you explain how these statements are being parsed?

Separately, I don't seriously want to support Windows, but is it okay to specify file paths using forward slashes when using C++ compilers on Windows machines?

Reply:
quotes vs brackets change how the pre-processor searches for included files. with quotes it will look relative to the current directory first, but it will still fallback to the usual search if that fails. for that reason using quotes usually doesn't change much. 

Using quotes is a convention to signal to the reader that "this header is one of ours".  I think it's helpful do adopt such conventions it makes it easier to differentiate our code from our dependencies, for example `#include <blas.hh>` for the BLAS++ header vs `#include "blaspp.h"` for our header.

the first line works because the path is actually correct, the second line works because, the file is found in one of the paths we told the pre-processor to look in via a command line argument. I thought about change the first line, but wanted to limit scope.

 yes, microsoft's  c pre-processor can handle /, what a mess life would be if it didn't!

Reply:
Gotcha. Makes sense. Thanks!